docs/articles/new-expensify/expenses-&-payments/Send-an-invoice.md

```diff
@@ -38,6 +38,9 @@ Once an invoice is sent, the customer receives an automated email or text messag
 4. Click **Pay Elsewhere**, which will mark the invoice as Paid. 
 
 Currently, invoices must be paid outside of Expensify. However, the ability to make payments through Expensify is coming soon. 
+
+![A photo of the pay button]({{site.url}}/assets/images/ExpensifyHelp-Invoice-1){:width="100%"}
```

Comment:
@Christinadobrzyn we forgot to add the file type 
```suggestion
![A photo of the pay button]({{site.url}}/assets/images/ExpensifyHelp-Invoice-1.png){:width="100%"}
```

Reply:
oh thank you for catching that @rushatgabhane! Let me know if you need me to take any action on my side.

Reply:
@Christinadobrzyn could you please commit this change

Reply:
@Christinadobrzyn You just need to click the `Commit suggestion` above. Technically @rushatgabhane or I could do it, but it keeps the separation of concerns when the author (not a reviewer) makes the commit.

![Screenshot 2024-07-11 at 8 20 09 AM](https://github.com/Expensify/App/assets/766197/f7d6a2db-5c18-4f33-ada3-944421bdc890)